src/WebJobs.Script/Binding/CoreExtensionsScriptBindingProvider.cs

```diff
@@ -81,7 +81,7 @@ public override Collection<Attribute> GetAttributes()
                     };
                     if (CrontabSchedule.TryParse(schedule, options) == null)
                     {
-                        throw new ArgumentException(string.Format("'{0}' is not a valid CRON expression.", schedule));
+                        throw new ArgumentException(string.Format("'{0}' is not a valid CRON expression. Schedule expressions in the form HH:MM:SS can only be used in an App Service Plan.", schedule));
```

Comment:
This isn't correct. E.g. if someone gives a malformed cron expression "* /30 *" you're saying that they specified a timespan format.

Why do we need to change anything here? When I added this code, I generalized the message so it indicated that only CRON expressions are allowed. The error message already says that.